libs/core/src/lib/split-button/split-button.component.ts

```diff
@@ -79,9 +81,22 @@ export class SplitButtonComponent {
     @ContentChild(MenuComponent)
     menu: MenuComponent;
 
+    constructor(private _changeDetectorRef: ChangeDetectorRef) { }
+
     /** @hidden Emits event when main button is clicked */
     onMainButtonClick(event: MouseEvent): void {
         this.primaryButtonClicked.emit(event);
         event.stopPropagation();
     }
+
+    ngAfterContentChecked(): void {
+        if (!this.mainActionTitle) {
```

Comment:
- Should be moved to separate method
- Never unsubscribed

Reply:
fixed